config/HELeptonInteractionListGenerator.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+
+<alg_conf>
+
+<!--
+Configuration for the HELeptonInteractionListGenerator InteractionListGeneratorI
+
+Configurable Parameters:
+.......................................................................................................
+Name             Type     Optional   Comment               Default
+.......................................................................................................
+-->
+
+  <param_set name="GLRES-Mu-Default">
+        <param type="bool" name="is-GLRES-Mu">          true  </param>
+  </param_set>
+
+  <param_set name="GLRES-Tau-Default">
+        <param type="bool" name="is-GLRES-Tau">         true  </param>
+  </param_set>
+
+  <param_set name="GLRES-Ele-Default">
+        <param type="bool" name="is-GLRES-Ele">         true  </param>
+  </param_set>
+
+  <param_set name="GLRES-Had-Default">
+        <param type="bool" name="is-GLRES-Had">         true  </param>
+  </param_set>
+
+  <param_set name="HENuEl-CC-Default">
+        <param type="bool" name="is-HENuEl-CC">         true  </param>
+  </param_set>
+
+  <param_set name="HENuEl-NC-Default">
+        <param type="bool" name="is-HENuEl-NC">         true  </param>
+  </param_set>
+
+  <param_set name="PhotonRES-Mu-Default">
+        <param type="bool" name="is-PhotonRES-Mu">      true  </param>
+  </param_set>
+
+  <param_set name="PhotonRES-Tau-Default">
+        <param type="bool" name="is-PhotonRES-Tau">     true  </param>
+  </param_set>
+
+  <param_set name="PhotonRES-Ele-Default">
+        <param type="bool" name="is-PhotonRES-Ele">     true  </param>
+  </param_set>
+
+  <param_set name="PhotonRES-Had-Default">
+        <param type="bool" name="is-PhotonRES-Had">     true  </param>
+  </param_set>
+
+  <param_set name="PhotonCOH-Default">
+        <param type="bool" name="is-PhotonCOH">         true  </param>
+  </param_set>
+
+</alg_conf>
+
```

Comment:
Do we have some kind of documentation of all the channels simulated in the HEDIS package now. This whole thing evolved to a substantial package with a lot of simulation capabilities. Can you consider some contribution to the physics section of the manual?

Reply:
This is a good point. So far, we explain these models in the following papers:

https://arxiv.org/abs/1908.10077
https://arxiv.org/abs/2004.04756

I think it is a good idea to include a section in the manual explaining how to run them. As soon as I have a bit of time, I will make this new section.